editor/src/components/inspector/sections/style-section/style-section.tsx

```diff
@@ -8,6 +8,14 @@ import { ShadowSubsection } from './shadow-subsection/shadow-subsection'
 import { TextShadowSubsection } from './text-subsection/text-shadow-subsection'
 import { AutosizingTextSubsection } from './text-subsection/text-subsection'
 import { TransformSubsection } from './transform-subsection/transform-subsection'
+import {
```

Comment:
it seems like you don't need these imports – probably leftover from previous commits

Reply:
Done!